basic/exercise5/script5.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-function firstSunday (startYear , endYear){
-    for (var year = startYear ; year <= endYear;  year++) {
+function firstSunday(startYear, endYear) {
```

Comment:
Function name is wrong - this function does not return first sunday as it is called. For now it returns first year from specifed range where 1st of january is Sunday.